tests/testpage.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
This is great for manual testing, but eventually we'll want to run automated tests in a test suite so we can develop at ease without risking things to break.

Nothing you need to worry about right now though. Just a note :)

Reply:
Yup that makes sense! I documented this in https://github.com/mozilla/sanitizer-polyfill/issues/10 and included info from our past conversations :)

Reply:
sure, yep